main_test.go

```diff
@@ -162,3 +162,91 @@ func TestAddPath(t *testing.T) {
 	expected := []string{"existing", "new"}
 	assert.Equal(t, expected, addPath(paths, "new"))
 }
+
+func TestLinterFlags(t *testing.T) {
```

Comment:
Renamed as suggested to align with the codebase. 

Yep it's a stylistic choice. For black-box testing I prefer naming tests based on functionality they're testing instead of the functions they're testing.